### ISSUE TITLE
Is updating file attributes in ACP supposed to work? it doesn't

### ISSUE THREAD

**Issue by gillyspy** [oracle/netsuite-suitecloud-sdk#867]
### Select the SuiteCloud tool where you encountered the issue

SuiteCloud CLI

### Version of the SuiteCloud Tool

2.0

### Operating system

Sequoia 15.1.1

### Node Version / VSCode Version

v18.15

### JAVA Version

Java(TM) SE Runtime Environment (build 17.0.6+9-LTS-190)

### Bug description

Trying to update file properties (description specifically) in an ACP. 

cannot get that to work.  
ref: https://docs.oracle.com/en/cloud/saas/netsuite/nsoa-online-help/section_4847748457.html

### Steps To Reproduce

1. create file
2. import it with `suitecloud file:import --paths /SuiteScripts/GG/whatever.js`
3. create deploy.xml
4. create manifest.xml
5. create project.json

deploy.xml looks like this:

```xml
<deploy>
<files>
  <path>~/FileCabinet/SuiteScripts/GG/whatever.js</path>
   <!-- works fine if i remove the next line -->
    <path>~/FileCabinet/SuiteScripts/GG/.attributes/whatever.js.attr.xml</path>
</files>
</deploy>
```

I've tried all kinds of permutations for deploy.xml.  This is what i was expecting would work

### Actual Output

importing it yeilded this: 
```txt
The imported files will overwrite the project files
The following files were imported:
/SuiteScripts/GG/.attributes/whatever.js.attr.xml
/SuiteScripts/GG/.attributes/.folder.attr.xml
/SuiteScripts/GG/whatever.js
```

Trying to deploy it with the properties yields this 
```txt
The deployment process has encountered an error.
Deploying to 123456_SB1
*** ERROR ***

Validation failed.

An error occurred during deploy file validation.
Details: The ~/FileCabinet/SuiteScripts/GG/.attributes/whatever.js.attr.xml file path must not be under the attributes folder.
File: ~/deploy.xml

2024-12-30 11:02:54 (PST) Installation FAILED (0 minutes 0 seconds)
```

### Expected Output

success

**Comment by gillyspy** [comment]
files that were created by file:import

```text
FileCabinet/SuiteScripts/GG
├── .attributes
│   ├── .folder.attr.xml
│   └── whatever.js.attr.xml
└── whatever.js
```

**Comment by alisyed-19** [comment]
@gillyspy Are you trying to update specific files and their properties instead of the whole project?
I suppose thats why you have modified `<path>~/FileCabinet/SuiteScripts/*</path>` ?

Have you tried the following?
```
<deploy>
<files>
  <path>~/FileCabinet/SuiteScripts/GG/*</path>
</files>
</deploy>
```

**Comment by gillyspy** [comment]
> @gillyspy Are you trying to update specific files and their properties instead of the whole project? I suppose thats why you have modified `<path>~/FileCabinet/SuiteScripts/*</path>` ?
> 
> Have you tried the following?
> 
> ```
> <deploy>
> <files>
>   <path>~/FileCabinet/SuiteScripts/GG/*</path>
> </files>
> </deploy>
> ```

yes updating specifics files as needed. the xml (would be) generated. if updating file attributes worked by `file:upload` that would be adequate but there are some features of `project:deploy` that I prefer for this scenario. 

I'll give that a try but the weakness is that the folder could contain more than one file and if this works i would be forced to make a tradeoff -- i.e. it still doesn't work but i get to pick between evils.  Will give it a shot

**Comment by alisyed-19** [comment]
@gillyspy Any update on this? the workaround was ok?
